README.md

```diff
@@ -252,6 +255,40 @@ TODO:
+
+- `csp=""`, for triggering [CSP Embedded Enforcement](https://w3c.github.io/webappsec-cspee/)
+- `policy=""`, for triggering [Document Policy](https://w3c.github.io/webappsec-permissions-policy/document-policy.html)
+- `sandbox=""`, for triggering [sandboxing](https://html.spec.whatwg.org/multipage/origin.html#sandboxing)
+- `allow=""`/`allowfullscreen=""`/`allowpaymentrequest=""`, for triggering [Permissions Policy](https://w3c.github.io/webappsec-permissions-policy/)
+
+This is a design choice based on the fact that portaling a page is more like linking to it than it is like embedding it, as discussed [below](#summary-of-differences-between-portals-and-iframes).
+
+In particular, after portal activation, it doesn't make sense for the host page to impose policies or delegate permissions. At that point the portaled content has become a full top-level browsing context, out of the original host's control. It might even navigate to a completely unrelated site, e.g. through the user clicking on an outgoing link.
+
+So, any mechanism for supporting this kind of embedder control or delegation would need to switch off upon activation. But these features aren't designed to do that; they all are imposed for the entire lifetime of the browsing context. We could try to create variants of them that only lasted for a document's lifetime, instead of an entire browsing context's lifetime, but this would pile confusion on top of an already-complicated space.
+
+An additional reason for avoiding these mechanisms is that it makes writing portalable content even harder. Not only would the content author have to deal with browser-imposed pre-activation [communications and storage restrictions](https://github.com/WICG/portals#privacy-threat-model-and-restrictions) and [restrictions to prevent it from acting outside of its rendered area](https://github.com/WICG/portals#other-restrictions-while-portaled)—it would also have to deal with embedder-specific restrictions, which could vary from embedder to embedder. Since, unlike iframes, portaled content generally wants to be portaled by many different embedders (e.g. different content aggregators all using portals for prerendering), this kind of ecosystem fragmentation is undesirable.
+
+To conclude, instead of giving embedders this control as iframes do, we believe that the browser can take the role of mitigating any problematic features. For example, instead of requiring embedders to use `sandbox=""` to turn off modal `alert()`/`confirm()`/`prompt()` dialogs, or permissions policy to turn off autoplaying media, those features are [always disabled](https://github.com/WICG/portals#other-restrictions-while-portaled) in pre-activation portals. And because portals are isolated from communicating with their embedder pre-activation, any problems which CSP Embedded Enforcement would attempt to protect against will instead be caught by this communications barrier and prevented from impacting the embedder.
```

Comment:
I think the behaviour for features controlled by permissions policy and permissions api is well defined (for existing features and for when we add any new features / change defaults), but I'm not super clear about portals do for the different `sandbox=` attributes. IIUC, we handle each attribute on a case by case basis, some make sense to enforce for portals, and some don't. If that's true, maybe it's worth adding a TODO to elaborate on the different sandbox attributes and which ones portals will be enforcing.

Reply:
Although, now that I actually look through all the possible sandbox values, I think `allow-forms` is the only behaviour that isn't really called out/explained in the explainer.

Reply:
Right, I did a more full analysis in https://github.com/WICG/portals/issues/237. My conclusion was

> If we want to add controls for preventing form submission, plugin usage, scripting, document.domain, or autoplay/autofocus, we use document policy for that in the future.

although I realize now autoplay is controlled by permissions policy that case in particular is already prevented.

Reply:
Hmm, although I guess the latest direction is to not allow document policy. Let's continue discussion in #237.